### DISCUSSION TITLE
Question about contribution

### DISCUSSION THREAD

**Discussion by VaibhavDS19** [gotham-rs/gotham#564] [General]
Hi guys, I am new to the open-source contribution side (have used a few open-source stuff for 2-3 years, two bugs found in some crates in the past 1 year), I am looking to learn and contribute more. Any suggestions on which issue to start with? I can contribute about 5-10 hours weekly for now (college drains a lot of time)? Thanks.

**Comment by msrd0** [comment]
Hi VaibhavDS19,

Great to hear that you want to contribute! Feel free to take a look at our issue tracker to find something to work on, but you are of course more than welcome to have ideas yourself. If you have any questions, feel free to ask them in our [Gitter](https://gitter.im/gotham-rs/gotham)/[Matrix](https://matrix.to/#/#gotham-rs_gotham:gitter.im) chatroom.
